client/helpers.go

```diff
@@ -237,12 +237,12 @@ func getRemoteKey(role data.RoleName, remote store.RemoteStore) (data.PublicKey,
 func rotateRemoteKey(role data.RoleName, remote store.RemoteStore) (data.PublicKey, error) {
 	rawPubKey, err := remote.RotateKey(role)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("unable to rotate remote key: %s", err)
```

Comment:
I've changed `%s` to `%v` but left the message intact because I had to reduce the cyclomatic complexity of `pubKeyListForRotation` by removing the error handling logic. 

https://github.com/docker/notary/pull/1147/commits/61f6fbe9b7a73842316a6c34164147a211f31d99#diff-1cd41a2611da49ce63630d5a51bd82c5L1162